packages/cubejs-schema-compiler/src/adapter/BaseQuery.js

```diff
@@ -278,11 +278,9 @@ export class BaseQuery {
       return dimension;
     }).filter(R.identity).map(this.newTimeDimension.bind(this));
     this.allFilters = this.timeDimensions.concat(this.segments).concat(this.filters);
+    this.useNativeSqlPlanner = this.options.useNativeSqlPlanner ?? getEnv('nativeSqlPlanner');
```

Comment:
It would be good to add `.default('false')` to this env so the useNativeSqlPlanner can be always bool.

Reply:
done